src/partials/course.hbs

```diff
@@ -20,11 +20,11 @@
         <span class="price">{{#if price}}₹{{{price}}}{{/if}}</span>
         <p class="description">{{description.[0]}}</p>
         <p class="lead">{{{lectures}}}</p>
-        <a class="btn btn-white btn-filled btn-lg" href="{{#relativePath @file.path link}}{{/relativePath}}">{{#unless
```

Comment:
You can't delete this line completely. Please check if the app is in working condition or not. You've to add this line in case its an offline course

Reply:
witty123 added below pls review.